packages/Textarea/src/Textarea.js

```diff
@@ -55,18 +55,13 @@ function Textarea(props) {
     return function cleanup() {
       window.removeEventListener("resize", resize);
     };
-  });
+  }, []);
```

Comment:
As discussed in https://github.com/acl-services/paprika/pull/499/files#r417552311